src/MIDebugEngine/Engine.Impl/Variables.cs

```diff
@@ -431,7 +431,13 @@ private string StripFormatSpecifier(string exp, out string formatSpecifier)
             // TODO: could return '(T(*)[n])(exp)' but requires T
             var m = Regex.Match(trimmed, @"^\[?(\d+)\]?$");
             if (m.Success)
+            {
+                if (_engine.DebuggedProcess.MICommandFactory.Mode == MIMode.Gdb)
+                    return "*" + exp.Substring(0, lastComma) + "@" + trimmed;  // this does not work for lldb
```

Comment:
@gc46 should work something like that `(lldb) parray 10 ptr`
https://lldb.llvm.org/use/map.html

Reply:
We cannot use the print method of GDB or LLDB, MIEngine uses MI Commands to communicate well formed output.

However, LLDB support has been included in https://github.com/microsoft/MIEngine/pull/1345